### DISCUSSION TITLE
[Support] Tabs are indented too much within `.txt` `ReadMe`s.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [community/community#27516] [Accessibility] [answered]
To ensure that I am able to track this and because I was informed that I should post it via here, I shall duplicate what I sent via [`support.github.com/ticket/personal/0/1706221#tc-7780686741012`](https://support.github.com/ticket/personal/0/1706221#tc-7780686741012:~:text=the%20readme%20file%20that%20https%3A%2F%2Fgithub.com%2Fbeedellrokejulianlockhart%2Fawn%2Ftree%2Fpatch-1%20presents%20is%20significantly%20too%20indented%20because%20it%20utilizes%20tabs%20whose%20indentation%20appears%20to%20be%20impossible%20to%20configure%20outside%20the%20editor%2C%20which%20causes%20the%20text-file%20to%20be%20difficult%20to%20read%20outside%20the%20editor):

<blockQuote>

#### Description

The `ReadMe` file that ~~[`raw.githubusercontent.com/RokeJulianLockhart/awn/3b978e6659aa7394775c5cc7544b0031d686df1d/ReadMe.txt`](https://raw.githubusercontent.com/RokeJulianLockhart/awn/3b978e6659aa7394775c5cc7544b0031d686df1d/ReadMe.txt)~~ [`RokeJulianLockhart/awn/blob/3b978e6659aa7394775c5cc7544b0031d686df1d/ReadMe.txt#L6-L50`](https://github.com/RokeJulianLockhart/awn/blob/3b978e6659aa7394775c5cc7544b0031d686df1d/ReadMe.txt#L6-L50) presents is significantly too indented because it utilizes tabs whose indentation appears to be impossible to configure outside the editor, which causes the text-file to be difficult to read outside the editor.

This is problematic because it negates the advantage to accessibility that tabs provide by being configurably long, and consequently causes them to be relevantly worse than two spaces.

#### Reproduction Steps

1. Create a `.txt`. `ReadMe`.

1. Add tabulators.

#### Fault Type

Permanent – Occurring repetitively.

</blockQuote>

**Comment by radar-blip** [comment]
Your Readme files is a .txt file. Shouldn't it be Readme.md ? Maybe thats why it is rendering differently than expected.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/community/discussions/27516#discussioncomment-3273053

@linuxlawson, my choice is deliberate:

> Pedro
> GitHub Support
> Pedro commented [11 days ago](https://support.github.com/ticket/personal/0/1706221#tc-7872597620756)
>
> Hi BEEDELLROKEJULIANLOCKHART,
> Let me help you out here! It seems that changing the "README" file extension from .txt to .md would solve the issue.
> Also, could you expand on your use case for using a .txt extension for your "README" file?
> 
> > BEEDELLROKEJULIANLOCKHART commented [11 days ago](https://support.github.com/ticket/personal/0/1706221#tc-7874362143764)
> > I am thankful for your response and suggested solution.
> > 
> > However, my use-case is that I specifically do not want any of the text of the file to be interpreted as Markdown, because conversion to Markdown would cause me to lose much of my ability to format my read-me. (If you copy the file to a repository and rename it to .md rather than .txt, you shall observe what I refer to.)
> > 
> > However, even if my rationale were better served by Markdown, I believe that this feature would be beneficial for others, because I see no good reason for the read-me view to provide fewer accessibility features than the editor, considering that the former is probably more frequently observed than the latter.
> > 
> > > Pedro commented [11 days ago](https://support.github.com/ticket/personal/0/1706221#tc-7872597620756)
> > > Your ticket was automatically routed to our GitHub Support team, but our Product team is better suited to help you out. I've passed this information on to the product team for future consideration.
> > >
> > > For future reference, the best way to report any feature requests or feedback is directly through our feedback form:
> > > [Share feedback](https://github.com/github/feedback/discussions)

That is a modified transcript of https://support.github.com/ticket/personal/0/1706221.

**Reply by benjiallen** [reply]
@BEEDELLROKEJULIANLOCKHART Thanks for the feedback. I'm interested to understand how you are defining accessibility. The feature idea you have makes sense but I think it's more of a presentation preference than an accessibility issue.

I'm defining accessibility as "an issue which has an impact on people with disabilities".

- Can you help me understand how you're defining accessibility?
- If we share the same definition, can you help me make the connection between this feature request and accessibility?

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/community/discussions/27516#discussioncomment-3319958

@benjiallen, [for some, larger indents are much more easy to visually process than smaller ones.](https://adamtuttle.codes/blog/2021/tabs-vs-spaces-its-an-accessibility-issue/) That is noticeably not true for me, but definitely is for my father, whose sight is not brilliant. This means that his documentation uses spaces usually because of unfortunate lack of support via services such as GitHub for configurable indentation-width.

**Reply by benjiallen** [reply, marked as answer]
@BEEDELLROKEJULIANLOCKHART have you had any luck configuring tab size using the [appearance settings](https://github.com/settings/appearance#tab-size-heading)?

[Docs for appearance settings](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference).

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/community/discussions/27516#discussioncomment-3366657

~~@benjiallen, no. Have you somehow?~~

**Comment by cehfisher** [comment]
Hi @RokeJulianLockhart - thank you for providing this feedback! This issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/27516#discussioncomment-3366657). Could you please confirm and close out this issue? If it is not resolved, what additional changes are needed? Please supply detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)

**Reply by roke-julian-lockhart** [reply]
> https://github.com/orgs/community/discussions/27516#discussioncomment-9539786

Indeed, @cehfisher, https://github.com/RokeJulianLockhart/awn/blob/3b978e6659aa7394775c5cc7544b0031d686df1d/ReadMe.txt#L6-L50 now looks fine. Obviously, the cited https://raw.githubusercontent.com/RokeJulianLockhart/awn/3b978e6659aa7394775c5cc7544b0031d686df1d/ReadMe.txt shan't change due to those settings, but that's the purview of the browser. Thanks for asking. I've consequently marked https://github.com/orgs/community/discussions/27516#discussioncomment-3366657 as the solution.

**Reply by cehfisher** [reply]
Thank you for following up! Glad to know this setting can help people like your father and others with low visual acuity 🙌🏻
